dashboard/app/controllers/projects_controller.rb

```diff
@@ -27,6 +27,12 @@ class ProjectsController < ApplicationController
     gumball: {
       name: 'New Gumball Project'
     },
+    mc: {
+      name: 'New Minecraft Simple Project'
```

Comment:
Good catch, I'll update the project name and URLs.  This isn't public-facing, but the URLs will be.